src/CSV/CSVWriter.php

```diff
@@ -35,6 +35,7 @@ class CSVWriter
     protected $options = [
         'write_utf8_bom' => FALSE,
         'quote_headers' => TRUE,
+        'eol' => PHP_EOL,
```

Comment:
Not that it affects us, but I think we should explicitly default to "\n" so that it behaves the same on all platforms?

Reply:
The default at the moment is PHP_EOL at least for the `fputcsv` call, so this would keep it non-breaking.

Reply:
Is that right? https://www.php.net/manual/en/function.fputcsv.php seems to say it's "\n"

Reply:
Changed it. PHPstorm lies